### DISCUSSION TITLE
Mobile Number Pad for Desktop

### DISCUSSION THREAD

**Discussion by bobby-sills** [ScratchAddons/ScratchAddons#5396] [Ideas] [answered]
On iPads, when you click inside a number input, it shows a small number pad to match the block color. Sometimes I find it mildly annoying to have to switch between mouse and keyboard a lot, and would rather just use the mouse. I think this would be an easy feature to implement because scratch already has it, just not for desktops.

**Comment by WorldLanguages** [comment]
I don't think I was aware of this Scratch feature. Could you send some screenshots?

**Reply by Samq64** [reply]
![Screenshot_20221203-185444_Firefox](https://user-images.githubusercontent.com/81489795/205467271-86e73f28-06f8-4e14-9267-617b29dd83d3.jpg)

**Reply by WorldLanguages** [reply]
Wow! It would be interesting to find a way to show this on desktop too, as an addon that can be enabled.
Very useful, specially for laptops with touch screens.

**Reply by Secret-chest** [reply]
> laptops with touch screens.

Laptops with touch screens already show it.

**Comment by GarboMuffin** [comment, marked as answer]
https://github.com/ScratchAddons/ScratchAddons/pull/5399
